redwood-layout-composeui/src/main/kotlin/app/cash/redwood/layout/utils.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:JvmName("compose_utils")
```

Comment:
I suspect we should move this whole package into a `.composeui` subpackage (and the view stuff into a `.view` subpackage in its artifact).

Reply:
This can be a follow-up